### ISSUE TITLE
How to install

### ISSUE THREAD

**Issue by janstuemmel** [karulont/i3lock-blur#20]
I was wondering. Ther's no installation guide, i'am not a c freak, but i know there must be some kind og `configure` or `Makefile`. But they're named different, so i'cant execute any of `-./configure` or `make`...

**Comment by JanzenJohn** [comment]
Did you find how to do it ?

**Comment by janstuemmel** [comment]
nope xD

**Comment by JanzenJohn** [comment]
Yeah I went with i3lock-fancy, however it takes like 2seconds to lock after execution. Did you find something else maybe?

**Comment by janstuemmel** [comment]
i just use default `i3-lock`
